draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -843,9 +843,16 @@ this parameter is not safe to accept over an untrusted channel.
 There is no SvcParamKey corresponding to the Alt-Svc "ma" (max age) parameter.
 Instead, server operators SHOULD encode the expiration time in the DNS TTL.
 
-Some DNS caching systems incorrectly extend the lifetime of DNS
-records beyond the stated TTL.  Server operators MUST NOT rely on
-HTTPSSVC records expiring on time, and MAY shorten the TTL to compensate.
+The appropriate TTL value will typically be similar to the "ma" value
+used for Alt-Svc, but may vary depending on the desired efficiency and
+agility.  Some DNS caches incorrectly extend the lifetime of DNS
+records beyond the stated TTL, so server operators cannot rely on
+HTTPSSVC records expiring on time.  Shortening the TTL to compensate
+for incorrect caching is NOT RECOMMENDED, as this practice impairs the
+performance of correctly functioning caches and does not guarantee
+faster expiration from incorrect caches.  Instead, server operators
+SHOULD continue operating the endpoints in expired records until they
+observe that nearly all connections have migrated to the new endpoints.
 
```

Comment:
Do we want to call out ESNI keys as well?

"Instead, server operators SHOULD continue operating the endpoints listed in expired records until they
observe that nearly all existing and new connections have migrated to the new endpoints (and are using new ESNI keys if applicable)."

Reply:
I tried to capture this by changing the text to describe "configurations" instead of "endpoints".